addons/im_livechat/security/ir.model.access.csv

```diff
@@ -7,3 +7,13 @@ access_livechat_support_report_operator,im_livechat.report.operator,model_im_liv
 access_livechat_channel_rule,im_livechat.channel.rule,model_im_livechat_channel_rule,,1,0,0,0
 access_livechat_channel_rule_user,im_livechat.channel.rule,model_im_livechat_channel_rule,im_livechat_group_user,1,1,1,0
 access_livechat_channel_rule_manager,im_livechat.channel.rule,model_im_livechat_channel_rule,im_livechat_group_manager,1,1,1,1
+access_chatbot_script,chatbot.script,model_chatbot_script,,1,0,0,0
+access_chatbot_script_user,chatbot.script.user,model_chatbot_script,im_livechat_group_user,1,1,1,0
+access_chatbot_script_manager,chatbot.script.manager,model_chatbot_script,im_livechat_group_manager,1,1,1,1
+access_chatbot_script_step,chatbot.script.step,model_chatbot_script_step,,0,0,0,0
+access_chatbot_script_step_user,chatbot.script.step.user,model_chatbot_script_step,im_livechat_group_user,1,1,1,0
```

Comment:
Doesn't that mean users can't remove steps from scripts? Even though they can essentially corrupt the scripts, or just replace steps? That seems like a  unnecessary limitation no?